pkg/cmd/pipeline/start.go

```diff
@@ -131,11 +131,12 @@ like cat,foo,bar
 			if opt.UseParamDefaults && (opt.Last || opt.UsePipelineRun != "") {
 				return errors.New("cannot use --last or --use-pipelinerun options with --use-param-defaults option")
 			}
-			if opt.DryRun {
```

Comment:
We are removing an option isnt it ? Maybe user would depend on this option and this update would break them, what do you think?

Reply:
Actually, we don't remove `dry-run` option and removing `opt.DryRun` block will not break current dry-run behavior. Only change I made is renaming `pipelineRunDryRun` function to `printPipelineRun` for reusing this function for non dry-run operations. ([diff](https://github.com/tektoncd/cli/pull/1326/files#diff-6d1b1e823072a56d8d9420b06b3b576d51901a18a703040a03c85fb10ae7f4e6L316-L318))

![image](https://user-images.githubusercontent.com/6194958/118924076-89dad980-b977-11eb-88dc-e725647b0d9d.png)